src/ctap/crypto_wrapper.rs

```diff
@@ -302,66 +250,30 @@ pub fn encrypt_key_handle(
 /// - the format does not match any known versions,
 /// - the HMAC test fails or
 /// - the relying party does not match the decrypted relying party ID hash.
-///
-/// This functions reads:
-/// - legacy credentials (no version number),
-/// - v1 (ECDSA)
-/// - v2 (EdDSA over curve Ed25519)
+/// - check_cred_protect is set and the credential requires user verification.
 pub fn decrypt_credential_source(
     env: &mut impl Env,
-    credential_id: Vec<u8>,
+    credential_id_bytes: Vec<u8>,
```

Comment:
This name does not match the specification.

Reply:
Done, and changed encrypt_to_bytes/decrypt_from_bytes to just encrypt/decrypt